alacritty/src/display.rs

```diff
@@ -118,6 +122,10 @@ pub struct Display {
     pub size_info: SizeInfo,
     pub font_size: Size,
     pub window: Window,
+    pub urls: Urls,
+
+    /// Currently highlighted URL.
+    pub url_highlighted: Option<Url>,
```

Comment:
`url_highlighted` -> `highlighted_url`/`active_url`? It's either a comment a bit misleading to me or a name.

Reply:
This was previously a boolean field, where `url_highlighted` makes a bit more sense. I think `highlighted_url` is better now, thanks.